src/main/java/com/paneedah/mwc/items/guns/M9A1Factory.java

```diff
@@ -21,588 +17,489 @@
 import net.minecraft.util.math.Vec3d;
 import org.lwjgl.opengl.GL11;
 
-import java.util.Arrays;
-
 public class M9A1Factory implements GunFactory {
 
     public Item createGun(CommonProxy commonProxy) {
         return new Weapon.Builder()
+                .withName("m9a1")
+                .withFireRate(0.5f)
+                .withRecoil(5f)
+                .withZoom(0.9f)
+                .withConfigGroup(GunConfigurationGroup.HANDGUN)
+                .withMuzzlePosition(new Vec3d(-.15, -0.8, -2.4))
+                .withShellType(Type.PISTOL)
+                .withMaxShots(1)
+                .withShootSound("m9a1")
+                .withSilencedShootSound("m9a1_silenced")
+                .withReloadSound("m9a1_reload")
+                .withUnloadSound("pistol_unload")
+                .withInspectSound("inspection")
+                .withDrawSound("handgun_draw")
+                .withReloadingTime(40)
+                .withFlashIntensity(0.5f)
+                .withFlashScale(() -> 0.6f)
+                .withFlashOffsetX(() -> 0.13f)
+                .withFlashOffsetY(() -> 0.14f)
+                .withInaccuracy(2)
```

Comment:
No balance changes until 0.2.
```suggestion
                .withInaccuracy(3)
```